e2e/test/scenarios/visualizations-charts/line-bar-tooltips.cy.spec.js

```diff
@@ -762,15 +729,21 @@ function setupDashboard(cardId, addedSeriesCardId) {
   });
 }
 
-function showTooltipForCircleInSeries(seriesColor, index = 0) {
-  cartesianChartCircleWithColor(seriesColor).eq(index).realHover();
+function resetHoverState() {
+  echartsTriggerBlur();
+  cy.get(POPOVER_ELEMENT).should("not.exist");
+  cy.wait(50);
```

Comment:
Without this line it has become way better but still flaked 1 out of 5 runs sometimes